debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -699,6 +701,17 @@ public static EventDeserializationFailureHandlingMode parse(String value) {
                                                                          .withDefault(1_000)
                                                                          .withValidation(Field::isNonNegativeLong);
 
+    public static final Field BUFFER_SIZE_FOR_BINLOG_READER = Field.create("binlog.buffer.size")
+                                                                   .withDisplayName("Binlog reader buffer size")
+                                                                   .withType(Type.INT)
+                                                                   .withWidth(Width.MEDIUM)
+                                                                   .withImportance(Importance.MEDIUM)
+                                                                   .withDescription("The size of a look-ahead buffer used by the  binlog reader to decide whether"
+                                                                           + "the transaction in progress is going to be committed or rolled back. "
+                                                                           + "Defaults to " + DEFAULT_BINLOG_BUFFER_SIZE + ". Use 0 to disable look-ahead buffering.")
```

Comment:
Hey @jpechane, WDYT about disabling the buffering in 0.7 by default? That way Shlomi and his team could gather some experiences with it, while nothing changes for others for now. And then we can enable it by default in a subsequent release. I think I'd feel a bit better about this change with such approach (want do to the same for DBZ-175 btw.)

Reply:
Yes, good idea!

Reply:
Disabled by default

Reply:
Hum, why didn't you adjust `DEFAULT_BINLOG_BUFFER_SIZE`? I'll change it.

Reply:
I wanted to keep it tere as a guideline to what we suppose to be a future default.